apiary-metastore-events/kafka-metastore-events/kafka-metastore-listener/pom.xml

```diff
@@ -43,6 +43,15 @@
                                     <include>org.apache.kafka:*</include>
                                 </includes>
                             </artifactSet>
+                            <relocations combine.children="append">
+                                <relocation>
+                                    <pattern>org.apache.kafka</pattern>
+                                    <shadedPattern>com.expediagroup.apiary.extensions.shaded.org.apache.kafka</shadedPattern>
```

Comment:
Is there no property for this: 'com.expediagroup.apiary.extensions.shaded.'

Reply:
We don't currently have a default `shade.prefix` or anything like that in the eg-oss-parent pom. We could add one if we feel it would be generally useful though?